src/getTicks.cpp

```diff
@@ -43,6 +43,8 @@
  * IN THE SOFTWARE.
  */
 
+#include <vector>
+#include <string>
```

Comment:
I think we get both of those via Rcpp.h as well, but doesn't hurt to be explicit.